CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
  - Clarify documentation about macro indirection
  - Added changelog
  - Turn the crate into a thin stdlib wrapper on rustc>=1.77
- - Remove `unstable_offset_of`
+ - Remove `unstable_offset_of` and `unstable_const`; they are not needed any more on recent nightlies
```

Comment:
I guess instead of entirely removing the features we could make them NOPs, then people don't have to update their Cargo.toml...

Reply:
I did that now.